### DISCUSSION TITLE
[1222680]极品飞车：热度

### DISCUSSION THREAD

**Discussion by JamesGeorgbb** [pjy612/SteamManifestCache#212] [Share]
`appid:1222680`
keys:
```
1222681;e249f05202889ee5fb53d5acc1e08682922c11271482791e5da05e6734701886
1239384;152c66145c46cb792b8b3a0e7b18f8336892ccd1e72bb0726039bdd843d40163
```
code:A03F98B881D36D5A
email:[976875393@qq.com](mailto:976875393@qq.com)
[1222680_config.vdf.zip](https://github.com/pjy612/SteamManifestCache/files/13797828/1222680_config.vdf.zip)
大佬这格式对吗

**Comment by cuteribs** [comment]
要登录EA, 怎么破呢
